ProcessLib/LiquidFlow/LiquidFlowProcess.h

```diff
@@ -72,6 +77,14 @@ class LiquidFlowProcess final : public Process
                                           GlobalVector const& x) override;
 
     bool isLinear() const override { return true; }
+
+    std::vector<MaterialLib::Fluid::FluidProperties*> getFluidProperties() const
+    {
+        std::vector<MaterialLib::Fluid::FluidProperties*> material_properties =
+            {_material_properties->getFluidProperties()};
```

Comment:
With that curly braces, I want to return a vector with one entry.

Initially, I want to have this function for all flow processes for coupling computation. Now I changed the mind, and made this function be unique only for LiquidFlow and return only a pointer to  class FluidProperties.